docker-compose.yml

```diff
@@ -35,13 +29,15 @@ services:
       - "3051:3051" # JSON RPC WS port
     volumes:
       # Configs folder bind
-      - type: bind
-        source: ./volumes/zksync/env.env
-        target: /etc/env/dev.env
+      - zksync-config:/etc/env/
```

Comment:
so now env ends up in a different location locally, right?

please mention it in PR description

Reply:
Indeed, Docker now efficiently manages volumes internally, typically within a designated folder at /var/lib/docker/volumes, with the necessary permissions. However, when using a bind volume with a relative path, the correct functioning of this feature relies on the permissions of the installation directory, which might vary on different machines. To ensure consistent behavior, it's advisable to opt for Docker-managed volumes whenever possible, as they eliminate such dependency on external factors.